### PR TITLE
add k8s annotation for loadbalencer

### PR DESCRIPTION
**Pull request by tinque**
Add kubernetes ingress annotations for loadbalencer : 
- traefik.backend.loadbalancer.sticky : for sticky session (true or false)
- traefik.backend.loadbalancer.method : for loadbalencer method (wrr or drr)

Update #674 and #911

### CODE REVIEW

**Comment by tinque** [comment]
Do you know why the tests do not work ?

File: provider/kubernetes.go [hunk 1]
```diff
@@ -229,6 +230,25 @@ func equalPorts(servicePort v1.ServicePort, ingressPort intstr.IntOrString) bool
...
 
+func getLoadBalencer(annotations map[string]string) *types.LoadBalancer{
+	loadbalancer := types.LoadBalancer{
+		Sticky: false,
+		Method: "wrr",
+	}
+
+	if len(annotations) > 0 && annotations["traefik.backend.loadbalancer.sticky"] =="true" {
+		loadbalancer.Sticky = true
+		log.Info("Create sticky loadbalancer")
```
Comment (tinque) [inline provider/kubernetes.go]: Yes, done !

**Comment by errm** [comment]
Is there a unit test that we could add here?

**Comment by tinque** [comment]
Sorry, but i have no idea how works your unit test. If I have any time, i will try to do this ...

**Comment by tinque** [comment]
Done

**Comment by emilevauge** [comment]
@tinque sorry could you rebase again ?

**Comment by tinque** [comment]
@emilevauge : I can't because I have a conflict on `docs/toml.md` and Github says that I don't have any write access on your repo to resolve this conflict.

**Comment by tinque** [comment]
Done

**Comment by tinque** [comment]
@emilevauge : yes but on consul ?! I didn't change anything ...

**Comment by emilevauge** [comment]
@tinque tests are failing in `kubernetes_test.go`. It should be linked ;)

**Comment by tinque** [comment]
Ok I'have fix the unit tests ;)

**Comment by tinque** [comment]
@emilevauge : I'm sorry but I have no time at this moment :(

**Comment by emilevauge** [comment]
@tinque FYI, code freeze for v1.2 is fixed on January the 31th. If you want you PR to be merged in this version, we need your commits as soon as possible :)

**Comment by vdemeester** [comment]
See #1068 for latests work, closing this one

**Pull request closed without merging**